### PR TITLE
`Paywalls`: updated template names

### PR DESCRIPTION
**Pull request by NachoSoto**

### CODE REVIEW

File: Sources/Paywalls/PaywallTemplate.swift [hunk 1]
```diff
@@ -17,10 +17,10 @@ import Foundation
 public enum PaywallTemplate: String {
 
     // swiftlint:disable missing_docs
-    case onePackageStandard = "one_package_standard"
-    case multiPackageBold = "multi_package_bold"
-    case onePackageWithFeatures = "one_package_with_features"
-    case multiPackageHorizontal = "multi_package_horizontal"
+    case template1 = "1"
```
Comment (joshdholtz) [inline Sources/Paywalls/PaywallTemplate.swift:20]: 🐱

File: Sources/Paywalls/PaywallTemplate.swift [hunk 2]
```diff
@@ -17,10 +17,10 @@ import Foundation
...
+    case template2 = "2"
```
Comment (joshdholtz) [inline Sources/Paywalls/PaywallTemplate.swift:21]: 🐅

File: Sources/Paywalls/PaywallTemplate.swift [hunk 3]
```diff
@@ -17,10 +17,10 @@ import Foundation
...
+    case template3 = "3"
```
Comment (joshdholtz) [inline Sources/Paywalls/PaywallTemplate.swift:22]: 🦁

File: Sources/Paywalls/PaywallTemplate.swift [hunk 4]
```diff
@@ -17,10 +17,10 @@ import Foundation
...
+    case template4 = "4"
```
Comment (joshdholtz) [inline Sources/Paywalls/PaywallTemplate.swift:23]: 🐆

**Review by joshdholtz — APPROVED** [review]

File: Sources/Paywalls/PaywallTemplate.swift (hunk 4 above)
Comment (NachoSoto) [reply Sources/Paywalls/PaywallTemplate.swift:23]: lol

**Pull request merged**